coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1150,15 +1130,43 @@ func (s *server) notifyWorkspaceManualBuildFailed(ctx context.Context, workspace
 				"template_version_name":    templateVersion.Name,
 				"initiator":                build.InitiatorByUsername,
 				"workspace_owner_username": workspaceOwner.Username,
+				"workspace_build_number":   strconv.Itoa(int(build.BuildNumber)),
```

Comment:
Including this will mean that deduplication won't be possible, just so you're aware.